### DISCUSSION TITLE
I have made a modified edition for WSR / SAPI 5

### DISCUSSION THREAD

**Discussion by Domtaro** [jwebmeister/tacspeak#26] [Show and tell]
Hi, jwebmeister.
I'm Domtaro, Japanese. I enjoying to play the Ready or Not with Tacspeak, thank you.

Now, I forked your repositories and published modified edition as ['TacspeakJP'](https://github.com/Domtaro/tacspeakJP).
It works on WSR / SAPI 5. I named 'JP' because, in the first I aimed at Japanese language speech recognition.
But I guess It can works for the other languages suppurted by WSR / SAPI 5.
Please feel free to tell me if you have any probrems about that.

I would like to bring notice that and, express my thanks and respect to you.

**Comment by jwebmeister** [comment]
Awesome! No problems from me. Good work :)

I do have a backlog "to-do" item to port the Tacspeak relevant features (e.g. listen_key) to the other speech engines (e.g. WSR / SAPI, DNS, etc.) that can be used by Dragonfly, but it's going to be a while until I have time to investigate and implement it.  You can track the issue [here](https://github.com/jwebmeister/dragonfly/issues/4) if you're interested.

**Reply by Domtaro** [reply]
I'm glad to hear that!
I haven't watched that log yet, I will check that. Thank you for sharing!
